soc/silabs/silabs_siwx917/Kconfig

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2025 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_SILABS_SIWX91X
+if MBEDTLS
+
+config PSA_CRYPTO_SW_ONLY
+	bool "PSA Crypto software implementation only"
+	select WISECONNECT_NETWORK_STACK
+
+config PSA_CRYPTO_DRIVER_SILABS_SI91X
```

Comment:
We use use `SILABS_SIWX91X`.

Reply:
SI91X is used as a macro in the mbedtls content, this is just a Kconfig wrapper for it. Do we want it to match Zephyr naming even if the underlying option is differently named?

Reply:
I don't know. I tend to only consider Zephyr and fully ignore the naming of the HAL.

Reply:
Done